.github/workflows/assigntome-docathon.yml

```diff
@@ -30,6 +31,9 @@ jobs:
               const { Octokit } = require("@octokit/rest");
               const octokit = new Octokit({
                 auth: process.env.GITHUB_TOKEN,
+                request: {
+                  fetch: require('cross-fetch'),
+                },
               });
               const { data: issue } = await octokit.issues.get({
```

Comment:
is there a reason you cant just do `github.rest.issues.get( etc )` instead of making a new Octokit instance? I see that this is what they do in the examples for github-script